ibis/tests/backends.py

```diff
@@ -57,6 +57,14 @@ def batting(self):
     def awards_players(self):
         return self.connection.database().awards_players
 
+    @classmethod
+    def make_context(cls, params=None):
+        module_name = cls.module
```

Comment:
There is a property for getting the module's name: `self.name`
We should use either explicit module variables or a method which converts the classname.

Reply:
> We should use either explicit module variables or a method which converts the classname.

I'm not sure I follow this. Isn't `self.name` enough?